src/cascadia/QueryExtension/AzureLLMProvider.idl

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import "ILMProvider.idl";
+
+namespace Microsoft.Terminal.Query.Extension
+{
+    [default_interface] runtimeclass AzureLLMProvider : ILMProvider
+    {
+        AzureLLMProvider();
+    }
+
+    [default_interface] runtimeclass AzureResponse : IResponse
```

Comment:
this doesn't need to be in the idl file. just make a `winrt::implements<>` like `DebugTapConnection`

Reply:
Only put something in the IDL file if people outside the DLL need to know the explicit type of it, and need to be able to access it.

Only put a public constructor in if somebody outside the DLL should be able to create one.